packages/tools/devtools/devtools-core/api-report/devtools-core.alpha.api.md

```diff
@@ -13,9 +13,27 @@ export interface ContainerDevtoolsProps extends HasContainerKey {
 // @beta
 export type ContainerKey = string;
 
+// @alpha
+export interface ContainerRuntimeDevtoolsProps {
+    label?: string;
+    runtime: IContainerRuntime;
+}
+
 // @beta
 export function createDevtoolsLogger(baseLogger?: ITelemetryBaseLogger): IDevtoolsLogger;
 
+// @alpha
+export interface DecomposedContainer extends IEventProvider<IContainerEvents> {
```

Comment:
We shouldn't need to export this. This should be an implementation detail of the package.

Reply:
My bad 😞, forgot to push in this particular change.